crates/swc/tests/fixture/issues-8xxx/8210/output/1.js

```diff
@@ -5,7 +5,7 @@ const Component = ()=>{
             fileName: "$DIR/tests/fixture/issues-8xxx/8210/input/1.js",
             lineNumber: 2,
             columnNumber: 23
-        }, void 0),
+        }, this),
```

Comment:
https://www.typescriptlang.org/play/?jsx=5#code/PTAEAECsGcA8C5QCcCmBDAxgFwLQ1gCYoBuAUBgPYB20WoAwhQLYAO1KVdAvKABQCUoLgD5QAb1KgpyFFgCuSKnwA8LUFgAWASyoBzLmOVpQwYQF9hACRQAbGxWXAWw-qTNA

This is a bugfix. Based on TypeScript's output, the correct value should be `this`. 
Not sure why SWC was outputting `undefined`.